tensorflow/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/reshaping/ZeroPadding1D.kt

```diff
@@ -10,6 +10,11 @@ import org.tensorflow.Shape
 /**
  * Zero-padding layer for 1D input (e.g. audio).
  * This layer can add zeros in the rows of the audio tensor
+ *
+ * __Input shape:__  3D tensor with shape `(batch_size, axis_to_pad, features)`.
```

Comment:
`axis_to_pad` is a somewhat misleading name because an axis usually refers to the dimension's index, but here it refers to the size of the dimension. Just `dim` would be ok.